dcos-labs/ui-kit
# DCOS-60141: does not set position if dropdownable is not open

This is a performance improvement that should resolve [DCOS-60141](https://jira.mesosphere.com/browse/DCOS-60141) when ui-kit gets updated in that Kommander with these changes.

## Testing
- Run a build of ui-kit using this branch
- Copy the build from `ui-kit/dist/` to `{kommander root}/packages/ui-components/node_modules/@dcos/ui-kit/dist/`
- In Kommander, navigate to the clusters cards view and resize the window

The card layout should not feel as slow as before, and the FPS should remain steady when resizing.

## Screenshots
### Before
[Video](https://cl.ly/c47ebfc9e5cc)

DevTools perf:
<img width="1679" alt="Screen Shot 2019-10-21 at 1 44 14 PM" src="https://user-images.githubusercontent.com/2313998/67229635-b7f9df80-f409-11e9-95df-4c274fa0a4fa.png">

### After
[Video](https://cl.ly/01088df763b9)

DevTools perf
<img width="1680" alt="Screen Shot 2019-10-21 at 1 28 13 PM" src="https://user-images.githubusercontent.com/2313998/67229650-bfb98400-f409-11e9-94ee-7cc906b43ad2.png">
